source/19-Psalms.usfm.db

```diff
@@ -2821,7 +2821,7 @@
 \q2 whom you will make princes in all the land.
 \q
 \v 17 Your name will I celebrate world without end,
-\q2 so that nations shall praise you for ever and ever.
+\q2 so that nations shall praise you [us:forever|cth:for ever] and ever.
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".